local-cli/runIOS/runIOS.js

```diff
@@ -27,15 +27,24 @@ function runIOS(argv, config) {
 }
 
 function _runIOS(argv, config, resolve, reject) {
-  const args = parseCommandLine([{
-    command: 'simulator',
-    description: 'Explicitly set simulator to use',
-    type: 'string',
-    required: false,
-    default: 'iPhone 6',
-  }], argv);
+  const args = parseCommandLine([
+    {
+      command: 'simulator',
+      description: 'Explicitly set simulator to use',
+      type: 'string',
+      required: false,
+      default: 'iPhone 6',
+    },
+    {
+      command: 'ios-project-path',
```

Comment:
I think it should be `project-dir` instead since it's not really a path? Apart from that, looks good.